aura/strike_popover/strike_popoverHelper.js

```diff
@@ -34,6 +37,45 @@
         var variant = component.get('v.variant');
         var placement = component.get('v.placement');
 
+        // Check for auto placement
+        if(placement.startsWith('auto ')) {
+
+            // Remove auto placement to perform bounding checks on preferred placement
+            placement = placement.replace('auto ', '');
+
+            // Construct the window bounding box.
+
+            // The height of the sticky global header is not accessible in any way
+            // so it must be hardcoded. This may change during releases.
+            var globalHeaderHeight = 90;
```

Comment:
Unfortunately, the top of the viewport is blocked by the global header and there is no way to get the height of it. I can't imagine that Salesforce would change this number by a huge value but the existing 90px just ensures that a popover that would sit in that space instead gets auto-positioned below.